content/workshops/openshift_4_101/lab5-rollbacks.md

```diff
@@ -66,37 +68,59 @@ The node.js builder template creates a number of resources for you, but what we
 </blockquote>
 
 ```bash
-$ oc describe bc/dc-metro-map | grep -i webhook
+$ oc get bc/dc-metro-map -o yaml | grep generic-webhook
+        name: dc-metro-map-generic-webhook-secret
+```
+
+<blockquote>
+<i class="fa fa-terminal"></i> Now that you have the name of the secret, you can get its value:
+</blockquote>
+
+```bash
+$ oc get secrets dc-metro-map-generic-webhook-secret -o yaml | grep -i key                    
+  WebHookSecretKey: MGE4NWYyZGZjMzFjZjJhMg==
+```
+
+<blockquote>
+<i class="fa fa-terminal"></i> Last, we can get the URL for the webhook
+</blockquote>
+
+```bash
+$ oc describe bc/dc-metro-map | grep -i webhook                                               
+Webhook Generic:                                                                                  
+        URL:            https://api.alexocp43.redhatgov.io:6443/apis/build.openshift.io/v1/namespaces/demo-1/buildconfigs/dc-metro-map/webhooks/<secret>/generic                                    
+Webhook GitHub:                                                                                   
+        URL:    https://api.alexocp43.redhatgov.io:6443/apis/build.openshift.io/v1/namespaces/demo-1/buildconfigs/dc-metro-map/webhooks/<secret>/github
 ```
 
 <blockquote>
-Copy the Generic webhook to the clipboard
+Copy the Generic webhook to the clipboard, replacing <b>&ltsecret&gt</b> with the value of the secret that you looked up, earlier.
 </blockquote>
 
 {{% /panel %}}
 
 {{% panel "Web Console Steps" %}}
         
 <blockquote>
-Click on "Builds" and then click on "Builds"
+Click on "Builds" and then click on "Build Configs"
```

Comment:
Suggest not using "on" throughout.  Just - Click "Builds" and then click "Build Configs".